docker_test.go

```diff
@@ -697,6 +762,16 @@ func TestPostgres(t *testing.T) {
 	require.Equal(t, myEvent.count, "1")
 }
 
+// Verify Event in POSTGRES
+func TestReportingService(t *testing.T) {
```

Comment:
This test assumes it will run after something else. For data to be there. In this case, it makes sense to move this check in the same test that produces data.